Makefile

```diff
@@ -239,3 +239,40 @@ ready-pr: lint manifests resource-docs generate-go-client
 upgrade-dcl:
 	go get github.com/GoogleCloudPlatform/declarative-resource-client-library
 	make ensure
+
+# Build all binaries
+all-binary: cmd-config-connector deletiondefender cmd-manager recorder unmanageddetector webhook gke-addon-poststart operator-cmd-manager
+.PHONY: all-binary cmd-config-connector deletiondefender cmd-manager recorder unmanageddetector webhook gke-addon-poststart operator-cmd-manager
+
+# Build cmd/config-connector binary
+cmd-config-connector:
```

Comment:
Also worth mention that .PHONY is used to indicate that the target is not a "real" target, i.e. it does not correspond to an actual file you are trying to make. However in this case you are trying to make bin/config-connector. So you could change the target to bin/config-connector and then it would not need to be a phony target. The problem then is that running 'make bin/config-connector' would do nothing if bin/config-connector already exists and is at least as old as its dependencies. Its probably fine for now but if put the effort into tracking its dependencies we could get an optimization.

Reply:
This is true, but go at least claims to be at least as good at tracking dependencies as make (or bazel).  So `go build` should be fast if dependencies are unchanged.

Reply:
Fair and I did end with its fine for now :P